Data/fe.yaml

```diff
@@ -5,7 +5,7 @@ meta:
   comment: maybe there is something to say here
   last_updated: 2020-04-28
   references:
-    @article{curtiss2013unicorn,
+    '@article{curtiss2013unicorn,
```

Comment:
Are you sure, that one needs the quotation marks around the references?
I would prefer it without them, but I do not know if we have actually tested the yaml parser against that.

Reply:
Without them the parser didn't work, as it is expecting a string, would be nice to get rid of it

Reply:
okay let's check that

Reply:
The core pyyaml `load` function does not recognise the `@` as a valid symbol and thus fails. One would need to:

1) before loading the file find the `references` part and add the quotation marks, or
2) scan the file for the `data` part and only parse this with yaml and the rest manually
3) use another loader. maybe `https://yaml.readthedocs.io/en/latest/install.html` fixes this. not sure